src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/X509Certificate2.cs

```diff
@@ -1190,6 +1289,14 @@ Oids.EcPublicKey when IsECDiffieHellman(certificate) =>
                             [PemLabels.Pkcs8PrivateKey],
                             MLKem.ImportFromPem,
                             certificate.CopyWithPrivateKey),
+                    Oids.SlhDsaSha2_128s or Oids.SlhDsaShake128s or Oids.SlhDsaSha2_128f or Oids.SlhDsaShake128f or
```

Comment:
This (and the one below it) would also be ideal candidates for that Helper I suggested earlier.